sdk/messaging/azeventhubs/CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 1.1.1 (Unreleased)
-
-### Features Added
+## 1.2.0 (2024-05-07)
 
 ### Breaking Changes
 
+- Processor.Run() behavior has been clarified and returns errors and handles closing
+  properly. See "Bugs Fixed" for more details about specific changes.
+
 ### Bugs Fixed
 
-### Other Changes
+The Processor had incorrect behavior when Run had been stopped:
+- Run() now returns an explicit error when called more than once on a single 
```

Comment:
The change looks slightly different than what's described here. It will prevent calling `Run()` on a stopped `Processor` but I don't see it preventing multiple calls to `Run()` on a non-stopped `Processor`. Maybe I missed it?

Reply:
Ah, multiple, not concurrent (maybe `repeated` would be better). I can clarify.

Reply:
Also, I should prevent the multiple calls thing too.